src/main/java/mateacademy/internetshop/db/Storage.java

```diff
@@ -14,4 +14,24 @@
     public static final List<Bucket> buckets = new ArrayList<>();
     public static final List<User> users = new ArrayList<>();
     public static final List<Order> orders = new ArrayList<>();
+
+    /*static {
```

Comment:
@AndreyDMA please do not commit commented code

Reply:
ok, I've just miss it...